application/values.yaml

```diff
@@ -570,6 +570,40 @@ ingress:
     #   hosts:
     #     - chart-example.local
 
+httpRoute:
+  # -- (bool) Enable HTTPRoute (Gateway API).
+  # @section -- HTTPRoute Parameters
+  enabled: false
+  # -- (list) Parent references for the HTTPRoute.
+  # @section -- HTTPRoute Parameters
+  parentRefs:
+    # - name: my-gateway
+    #   sectionName: bar
+  # -- (string) Namespace of the Gateway to attach this HTTPRoute to.
+  # If not set, the HTTPRoute will be attached to the Gateway in the same namespace as the HTTPRoute.
+  # @section -- HTTPRoute Parameters
+  gatewayNamespace: ""
+  # -- (list) Hostnames for the HTTPRoute.
+  # @section -- HTTPRoute Parameters
+  hostnames:
+    - "chart-example.local"
+  # -- (object) Additional labels for HTTPRoute.
+  # @section -- HTTPRoute Parameters
+  additionalLabels: {}
+  # -- (object) Annotations for HTTPRoute.
+  # @section -- HTTPRoute Parameters
+  annotations: {}
+  # -- (list) Rules for HTTPRoute.
+  # @section -- HTTPRoute Parameters
+  rules:
+    - matches:
+        - path:
+            type: PathPrefix
+            value: /
+      backendRefs:
+        - name: http
+          port: 8080
```

Comment:
Can you please align this on the ingress implementation?

Reply:
What do you mean by  align?

Reply:
I meant align so their values looks like the same. The goal is to auto-link this rule to the application service.